src/server/routes/search.js

```diff
@@ -37,6 +37,10 @@ const SearchPage = React.createFactory(
 
 const router = express.Router();
 
+/**
+ * Returns React markup for the search page that is rendered by the user's
```

Comment:
Don't like the use of "Returns" here - "Generates" would be better to avoid confusion (it doesn't actually *return* anything!)